api/app/controllers/mno_enterprise/jpi/v1/app_answers_controller.rb

```diff
@@ -2,21 +2,20 @@ module MnoEnterprise
   class Jpi::V1::AppAnswersController < Jpi::V1::AppReviewsController
 
     private
-
-    def scope_app_reviews
-      @app_reviews.where(question_id: params[:question_id])
-    end
-
     def after_save
       #do nothing because answers doesn't affect on app rating
     end
 
     def review_klass
-      MnoEnterprise::AppAnswer
+      MnoEnterprise::Answer
+    end
+
+    def initial_scope
+      review_klass.where(parent_id: params[:parent_id])
```

Comment:
No need too, as it is going to be added in the `AppReviewsController` index method anyway.